docs/_includes/ww-admin-tasks-content.md

```diff
@@ -132,17 +142,27 @@ Staff users with the appropriate permissions may create and edit pre-written res
 
 Click on ‘Templates’ in the admin menu. You will see a table of any existing templates. Click on the button below, marked ‘New template’.
 
-Common template responses deal with statuses such as: Report received; Collection scheduled; Container ordered; Report under investigation.
+<img alt="New template button" src="/assets/img/ww-user-guide/WasteWorks new template button.png" />
+
+To create a template:
+1. Input a title. This should be something easy to understand like 'Bin not presented'.
+2. Write more detail in the 'Text' box. This will be displayed on the WasteWorks site to users viewing information on a property's collections.
+3. Also write some text in the 'Text for email alert field'. This text will be emailed to the user if they have opted into alerts.
+4. Set which categories the template should apply to. You may create different templates for each category, or use the same templates across multiple categories.
+5. Optionally assign a 'State' to the template if you want to use it to prefill the update field when changing a report’s state (e.g. 'investigating').
+6. Optionally provide an external status code if you want to use the template to prefill the update field when changing a report’s state.
+7. Optionally tick 'Auto response' if you want this template to be used for Open311 updates that put problems in this state.
```

Comment:
Open311 won't make much sense in a WW context (though it is), I'd use "updates from the backend"